client/src/components/CatalogMainSection/CatalogMainSection.test.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable react/react-in-jsx-scope */
+import { render } from '@testing-library/react';
+import CatalogMainSection from './CatalogMainSection';
+
+describe('CatalogMainSection snapshot testing', () => {
+  test('should CatalogMainSection match snapshot', () => {
+    const { asFragment } = render(<CatalogMainSection />);
+
+    expect(asFragment()).toMatchSnapshot();
+  });
+});
```

Comment:
please check other test files already created.   
you have to mock all MUI components.